DataDefinitions/ShipDefinitions.cs

```diff
@@ -36,14 +36,14 @@ public class ShipDefinitions
             { 128049285, new Ship(128049285, "Type6", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Type-6 Transporter", null, "Medium", null, 0.39M) },
             { 128049297, new Ship(128049297, "Type7", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Type-7 Transporter", null, "Large", null, 0.52M) },
             { 128049333, new Ship(128049333, "Type9", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Type-9 Heavy", null, "Large", null, 0.77M) },
-            { 128049273, new Ship(128049273, "Viper", "Faulcon DeLacy", null, "Viper Mk. III", new List<Translation> {new Translation("viper", "ˈvaɪpə"), new Translation("Mk.", "mɑːk"), new Translation("III", "θriː") }, "Small", 3, 0.41M) },
-            { 128672255, new Ship(128672255, "Viper_MkIV", "Faulcon DeLacy", null, "Viper Mk. IV", new List<Translation> {new Translation("viper", "ˈvaɪpə"), new Translation("Mk.", "mɑːk"), new Translation("IV", "fɔː") }, "Small", 3, 0.46M) },
+            { 128049273, new Ship(128049273, "Viper", "Faulcon DeLacy", null, "Viper Mk. III", new List<Translation> {new Translation("viper", "ˈvaɪpə"), new Translation("Mk", "mɑːk"), new Translation("III", "θriː") }, "Small", 3, 0.41M) },
+            { 128672255, new Ship(128672255, "Viper_MkIV", "Faulcon DeLacy", null, "Viper Mk. IV", new List<Translation> {new Translation("viper", "ˈvaɪpə"), new Translation("Mk", "mɑːk"), new Translation("IV", "fɔː") }, "Small", 3, 0.46M) },
             { 128049309, new Ship(128049309, "Vulture", "Core Dynamics", null, "Vulture", new List<Translation> { new Translation("vulture", "ˈvʌltʃə") }, "Small", 5, 0.57M) },
             { 128785619, new Ship(128785619, "Type9_Military", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Type-10 Defender", null, "Large", 5, 0.77M) },
             { 128816574, new Ship(128816574, "TypeX", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Alliance Chieftain", null, "Medium", 4, 0.77M) },
             { 128816581, new Ship(128816581, "TypeX_2", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Alliance Crusader", null, "Medium", 4, 0.77M) },
             { 128816588, new Ship(128816588, "TypeX_3", "Lakon Spaceways", new List<Translation> {new Translation("Lakon", "leɪkɒn") }, "Alliance Challenger", null, "Medium", 4, 0.77M) },
-            { 128816567, new Ship(128816567, "Krait_MkII", "Faulcon DeLacy", null, "Krait Mk. II", new List<Translation>{new Translation("Krait", "ˈkreɪt"), new Translation("Mk.", "mɑːk"), new Translation("II", "ˈtuː") }, "Medium", null, 0.63M) },
+            { 128816567, new Ship(128816567, "Krait_MkII", "Faulcon DeLacy", null, "Krait Mk. II", new List<Translation>{new Translation("Krait", "ˈkreɪt"), new Translation("Mk", "mɑːk"), new Translation("II", "ˈtuː") }, "Medium", null, 0.63M) },
```

Comment:
Does this still confuse non-SSML capable voices (since they still may pronouce "m k 3" rather than "mark 3")?

Since we're building both the IPA string and the non-IPA alternative, what if we were to do something like this?
```cs
            { 128816567, new Ship(128816567, "Krait_MkII", "Faulcon DeLacy", null, "Krait Mk. II", new List<Translation>{new Translation("Krait", "ˈkreɪt"), new Translation("Mark", "mɑːk"), new Translation("2", "ˈtuː") }, "Medium", null, 0.63M) },
```

Reply:
Yeah, they still say "em key three". 

> Since we're building both the IPA string and the non-IPA alternative, what if we were to do something like this?

This would indeed be a better (and a very obvious) thing to do. I made this particular commit back when I had very little idea about how translations work in EDDI...

Reply:
Commit superseded by PR #1618.